### ISSUE TITLE
Can not add Suggester

### ISSUE THREAD

**Issue by mathiso** [fortedigital/EpiServer.AzureSearch#29]
I have tried to find out how to add Suggester to fields when creating the index. Is that even possible?

**Comment by tomaszmadeyski** [comment]
No, currently we don't have support for Suggester, but this is a nice idea to add one!

**Comment by mathiso** [comment]
Okej, I understand! Is there any possible way to create the index before it is created by this plugin? Is there any event or something where we can handle our index creation?

**Comment by tomaszmadeyski** [comment]
I think you should give a try to `IIndexSpecificationProvider`. You can register your own implementation in DI container (by default it is `NullIndexSpecificationProvider`). In this specification you can perform any action that will setup your index when it is being created.
